ddtrace/contrib/django/utils.py

```diff
@@ -303,6 +305,27 @@ def _after_request_tags(pin, span, request, response):
             raw_uri = url
             if raw_uri and request.META.get("QUERY_STRING"):
                 raw_uri += "?" + request.META["QUERY_STRING"]
+
+            req_body = None
+
+            if config._appsec_enabled and request.method in _BODY_METHODS:
```

Comment:
From what I've seen the request.body and request.POST can return an exception, we may need to handle those case
You can see here how it was done at Sqreen:
https://github.com/sqreen/AgentPython/blob/develop/sqreen/frameworks/django_framework.py#L54
cc @nizox

Reply:
Broken link.

Reply:
As sorry it's a private repo, I'm giving you access!

Reply:
Added the exceptions that I could see it could raise